openpype/hosts/maya/api/lib.py

```diff
@@ -1530,9 +1530,9 @@ def get_container_members(container):
         reference_members = cmds.ls(reference_members,
                                     long=True,
                                     objectsOnly=True)
-        members.update(reference_members)
+        all_members.update(reference_members)
 
-    return members
+    return list(all_members)
```

Comment:
Seem to work now, tested in Maya2018.7.
Both asset loading and look assignment work!